.github/workflows/release.yml

```
@@ -0,0 +1,202 @@
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  androidx:
+    name: AndroidX
+    uses: ./.github/workflows/nuget-release.yml
+    needs: build
+    with:
+      tag-prefix: 'androidx-'
+      project-path: "src/Auth0.OidcClient.AndroidX"
+      nuspec-file: "nuget/Auth0.OidcClient.AndroidX.nuspec"
+    secrets:
+      nuget-token: ${{ secrets.NUGET_APIKEY }}
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  ios:
+    name: iOS
+    uses: ./.github/workflows/nuget-release.yml
+    needs: build
+    with:
+      tag-prefix: 'ios-'
+      project-path: "src/Auth0.OidcClient.iOS"
+      nuspec-file: "nuget/Auth0.OidcClient.iOS.nuspec"
+    secrets:
+      nuget-token: ${{ secrets.NUGET_APIKEY }}
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  core:
+    name: Core
+    uses: ./.github/workflows/nuget-release.yml
+    needs: build
+    with:
+      tag-prefix: 'core-'
+      project-path: "src/Auth0.OidcClient.Core"
+      nuspec-file: "nuget/Auth0.OidcClient.Core.nuspec"
+    secrets:
+      nuget-token: ${{ secrets.NUGET_APIKEY }}
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  wpf:
+    name: WPF
+    uses: ./.github/workflows/nuget-release.yml
+    needs: build
+    with:
+      tag-prefix: 'wpf-'
+      project-path: "src/Auth0.OidcClient.WPF"
+      nuspec-file: "nuget/Auth0.OidcClient.WPF.nuspec"
+    secrets:
+      nuget-token: ${{ secrets.NUGET_APIKEY }}
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+    
+  winforms:
+    name: WinForms
+    uses: ./.github/workflows/nuget-release.yml
+    needs: build
+    with:
+      tag-prefix: 'winforms-'
+      project-path: "src/Auth0.OidcClient.WinForms"
+      nuspec-file: "nuget/Auth0.OidcClient.WinForms.nuspec"
+    secrets:
+      nuget-token: ${{ secrets.NUGET_APIKEY }}
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  uwp:
+    name: UWP
+    uses: ./.github/workflows/nuget-release.yml
+    needs: build
+    with:
+      tag-prefix: 'uwp-'
+      project-path: "src/Auth0.OidcClient.UWP"
+      nuspec-file: "nuget/Auth0.OidcClient.UWP.nuspec"
+    secrets:
+      nuget-token: ${{ secrets.NUGET_APIKEY }}
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  maui:
+    name: MAUI
+    uses: ./.github/workflows/nuget-release.yml
+    needs: build
+    with:
+      tag-prefix: 'maui-'
+      project-path: "src/Auth0.OidcClient.MAUI"
+      nuspec-file: "nuget/Auth0.OidcClient.MAUI.nuspec"
+    secrets:
+      nuget-token: ${{ secrets.NUGET_APIKEY }}
+      github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  generate-docs:
+    name: Generate API docs
+    if: github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && github.event.pull_request.merged && startsWith(github.event.pull_request.head.ref, 'release/'))
+    runs-on: windows-latest
+    environment: release
+    needs: build
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/download-artifact@v4
```

Is it worth downgrading this to v3 as there seems to be issues amongst the latest *-artifact action releases

```suggestion
      - uses: actions/download-artifact@v3
```

---

Wont do this